localisation/000_wc_traits.csv

```diff
@@ -622,3 +622,9 @@ proud_desc;For some reason this character is proud or even selfish. Proud charac
+creature_wicker_golem;Wicker Golem;;Weidengolem;;;;;;;;;;;x
+creature_wicker_golem_desc;This character is a wicker golem. Wicker golems are created by Throsic covens from inanimate organic materials such as wood, flesh, fur, and bones and are animated by ancient Drust death magic. They are powered by the souls of mortal victims or by the spirits of ancient Drust and thus give the spirits physical form. Due to an excruciating soul-ripping process, most victims' souls are shattered to the point of losing all cognitive intelligence, although some of the souls remain aware of just how trapped they are within their wicker bodies.;;Dieser Charakter ist ein Weidengolem. Weidengolems werden von throsischen Zirkeln aus unbelebten organischen Materialien wie Holz, Fleisch, Fell und Knochen erschaffen und durch die antike Todesmagie des Drust animiert. Sie werden von den Seelen der sterblichen Opfer oder von den Geistern der antiken Drust angetrieben und geben den Geistern somit eine physische Form. Durch einen qualvollen, seelenzerst�renden Prozess werden die Seelen der meisten Opfer so weit zerrissen, dass sie jeden Rest ihrer geistigen Kraft verlieren, obwohl einige der Seelen sich bewusst bleiben, wie gefangen sie in ihren Weidenk�rpern sind.;;;;;;;;;;;x
+creature_wicker_behemoth;Wicker Behemoth;;Weidenunget�m;;;;;;;;;;;x
+creature_wicker_behemoth_desc;This character is a wicker behemoth. Wicker behemoths are created by Throsic covens from inanimate organic materials such as wood, flesh, fur, and bones and are animated by ancient Drust death magic. They are powered by the souls of mortal victims or by the spirits of ancient Drust and thus give the spirits physical form. The wicker behemoths combine mobility and physical strength, making them very efficient enforcers upon the front lines, bent on devouring the souls of the unfortunate victims that dare to cross their path.;;Dieser Charakter ist ein Weidenunget�m. Weidenunget�me werden von throsischen Zirkeln aus unbelebten organischen Materialien wie Holz, Fleisch, Fell und Knochen erschaffen und durch die antike Todesmagie des Drust animiert. Sie werden von den Seelen der sterblichen Opfer oder von den Geistern der antiken Drust angetrieben und geben den Geistern somit eine physische Form. Die Weidenunget�me vereinen Beweglichkeit und K�rperkraft, was sie zu sehr effizienten Streitern an der Front macht, die darauf aus sind, die Seelen der ungl�cklichen Opfer zu verschlingen, die es wagen, ihren Weg zu kreuzen.;;;;;;;;;;;x
```

Comment:
"making them very efficient enforcers upon the front lines," 
the 'efficient' could be changed with 'effective' 
also would sound nicer if 'upon' was changed to 'on'

Reply:
changed, need German corrections now.

Reply:
already translated it by meaning instead of word by word.